aiohttp/payload.py

```diff
@@ -87,12 +100,18 @@ def register(self, factory, type, *, order=Order.normal):
 
 class Payload(ABC):
 
-    _size = None
-    _headers = None
-    _content_type = 'application/octet-stream'
-
-    def __init__(self, value, *, headers=None, content_type=sentinel,
-                 filename=None, encoding=None, **kwargs):
+    _size = None  # type: Optional[float]
+    _headers = None  # type: Optional[LooseHeaders]
```

Comment:
`_headers` has the concrete type `Optional[CIMultiDict[str]]`.

`LooseHeaders` is used for passing any mapping-like type.
aiohttp client API converts `LooseHeaders` into `CIMultiDict` internally.

I'm not 100% sure but a payload should accept `CIMultiDict` objects only.

Reply:
@asvetlov Given [this](http://werkzeug.pocoo.org/docs/0.14/datastructures/) documentation, I think that it won't hurt to write typing for `__init__` to reflect all the possible types `CIMultiDict` can accept. I will push the next commit with that and you can check it out later on.